doc/coog_admin_tuto.rst

```diff
@@ -0,0 +1,676 @@
+
+Once the database is set, applicative servers can be run through the following
+commands
+
+.. code-block:: bash
+
+    ./coog server # Will launch Coog container
+    ./coog celery # Will launch Coog Celery
+    ./web server
+    ./nginx reset
+    ./nginx run
+
+It can happen that containers need to be restarted. In this case
+
+.. code-block:: bash
+
+    ./upgrade
+
+Containers and applicative servers can be stopped through the following commands
+
+.. code-block:: bash
+
+    ./redis rm -f
+    ./postgres rm -f
+    ./nginx rm -f
+    ./web rm -f
+    ./coog -- server rm -f
+    ./coog -- celery rm -f
+    ./coog -- cron rm -f
+    ./unoconv rm -f
+
+Test environment
+----------------
+
+The environment is ready to be tested.
+
+* Backoffice is accessible through http://hostname
+* Documentation is accessible through http://hostname/doc
+* Bench tool is accessible through http://hostname/bench
+* API REST is accessible through http://hostname/web/api
+* Modules selection application is accessible through
+  http://hostname/web/#install/start
+
+If you want to check API is working, launch a Get on
+http://hostname/web/api/auth
+check it returns
+
+.. code-block:: bash
+
+    {"ok": false}
+
+* Front office web app is available through http://hostname/web
+
+Batch
+-----
+
+The *batch* command allows executing a coog batch. A celery batch worker must
+be launched in order for it to work properly. Its execution follows the ordered
+steps:
+
+* Jobs generation
+* Batch execution
+* *Optional*: Failed batches split and wait for new jobs génération
+* Return with exit status *OK* if all jobs succeed
+
+The execution of a chain and of the daily chain follow the same routine. These
+commands are usually launched by **cron** and their outputs are usually
+configured to be sent by mail.
+
+This is an example of how to launch *Coog*'s *ir.ui.view.validate* batch:
+
+.. code-block:: bash
+
+   ./coog celery 1
+   ./coog batch ir.ui.view.validate --job_size=10
+   echo $?
+   ./coog redis celery qlist ir.ui.view.validate
+   ./coog batch ir.ui.vuew.validate --job_size=100 --crash=144
+   ./coog redis celery q ir.ui.view.validate 
+
+Here are some useful celery commands
+
+* For all queues:
+
+.. code-block:: bash
+
+    ./coog redis celery list
+    ./coog redis celery flist 
+
+* For one queue:
+
+.. code-block:: bash
+
+    ./coog redis celery fail ir.ui.view.validate
+    ./coog redis celery q ir.ui.view.validate 
+    ./coog redis celery qlist ir.ui.view.validate 
+    ./coog redis celery qcount ir.ui.view.validate 
+    ./coog redis celery qtime ir.ui.view.validate 
+    ./coog redis celery qarchive ir.ui.view.validate 
+    ./coog redis celery qremove ir.ui.view.validate 
+
+* For one job:
+
+.. code-block:: bash
+
+    ./coog redis celery j
+    ./coog redis celery jarchive
+    ./coog redis celery jremove
+
+**cron** configuration allows handling jobs execution generation and monitoring,
+and notifying batch chain execution end by email
```

Comment:
should not we have an example of cron config for this ?

Reply:
je n'en ai pas sous la main :'( mais preneuse si on m'en envoie une